dex/testing/eth/harness.sh

```diff
@@ -20,6 +20,8 @@ ALPHA_NODE_KEY="71d810d39333296b518c846a3e49eca55f998fd7994998bb3e5048567f2f073c
 ALPHA_ENODE="897c84f6e4f18195413c1d02927e6a4093f5e7574b52bdec6f20844c4f1f6dd3f16036a9e600bd8681ab50fd8dd144df4a6ba9dd8722bb578a86aaa8222c964f"
 ALPHA_NODE_PORT="30304"
 ALPHA_AUTHRPC_PORT="8552"
+ALPHA_HTTP_PORT="38556"
+ALPHA_WS_PORT="38557"
```

Comment:
Would like to move this to the full nodes, since that is what would be used irl.